src/main/java/com/cglx/Calculator.java

```diff
@@ -52,7 +52,7 @@ public int square(int a)
       }
       catch (Exception e)
       {
-        //System.err.format("Exception occurred trying to read '%s'.", filename);
+        System.err.format("Exception occurred trying to read '%s'.", filename);
         e.printStackTrace();
         return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01.ahprod.adhaven.com:9990/coding_rules#rule_key=squid%3AS1168)